JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -650,6 +650,12 @@ - (BOOL)collectionView:(JSQMessagesCollectionView *)collectionView shouldShowMen
     //  temporarily disable 'selectable' to prevent this issue
     JSQMessagesCollectionViewCell *selectedCell = (JSQMessagesCollectionViewCell *)[collectionView cellForItemAtIndexPath:indexPath];
     selectedCell.textView.selectable = NO;
+    
+    //  it will reset the font and fontcolor when selectable is NO
+    //  however, the actual font and fontcolor in textView do not get changed
+    //  in order to keep text like "$1200" to stick with the same style, we need to re-assign the font and fontcolor when selectable is NO
```

Comment:
last line:

> // in order to preserve link colors,  we need to re-assign the font and fontcolor when selectable is NO

let's also add the original GitHub issue number, plus this PR number:

> // See GitHub issues #1675 and #1759